src/flamegraph/color/palettes.rs

```diff
@@ -59,6 +59,21 @@ pub(super) mod perl {
     }
 }
 
+pub(super) mod python {
+    use crate::flamegraph::color::BasicPalette;
+
+    pub fn resolve(name: &str) -> BasicPalette {
+        if name.starts_with("native@") { // austin-specific format for native calls
+            return BasicPalette::Aqua;
+        } else if name.contains("site-packages/") {
```

Comment:
```suggestion
        } else if name.contains("/site-packages/") {
```
is probably more robust

Reply:
Fixed 👍